### ISSUE TITLE
Unable to install gpu-operator on kubernetes v1.17.5. cross-namespace owner references are disallowed, owner's namespace gpu-operator, obj's namespace gpu-operator-resources"

### ISSUE THREAD

**Issue by dkozlov** [NVIDIA/gpu-operator#62]
_The template below is mostly useful for bug reports and support questions. Feel free to remove anything which doesn't apply to you and add more information where it makes sense._

### 1. Quick Debug Checklist
- [x] Are you running on an Ubuntu 18.04 node?
- [x] Are you running Kubernetes v1.13+?
- [x] Are you running Docker (>= 18.06) or CRIO (>= 1.13+)?
- [x] Do you have `i2c_core` and `ipmi_msghandler` loaded on the nodes?
- [x] Did you apply the CRD (`kubectl describe clusterpolicies --all-namespaces`)

### 1. Issue or feature description

```
$kubectl logs pod/gpu-operator-6fd59b7c9d-6ccz4 -n gpu-operator
{"level":"error","ts":1589117072.6562517,"logger":"controller-runtime.controller","msg":"Reconciler` error","controller":"clusterpolicy-controller","request":"gpu-operator/cluster-policy","error":"cross-namespace owner references are disallowed, owner's namespace gpu-operator, obj's namespace gpu-operator-resources","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.5.2/pkg/internal/controller/controller.go:258\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.5.2/pkg/internal/controller/controller.go:232\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).worker\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.5.2/pkg/internal/controller/controller.go:211\nk8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\t/go/pkg/mod/k8s.io/apimachinery@v0.17.4/pkg/util/wait/wait.go:152\nk8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/pkg/mod/k8s.io/apimachinery@v0.17.4/pkg/util/wait/wait.go:153\nk8s.io/apimachinery/pkg/util/wait.Until\n\t/go/pkg/mod/k8s.io/apimachinery@v0.17.4/pkg/util/wait/wait.go:88"}
```

### 2. Steps to reproduce the issue

Create kubernetes cluster v1.17.5. Install gpu-operator-1.1.6   by helm chart, check the gpu-operator logs by ```kubectl logs pod/gpu-operator-6fd59b7c9d-6ccz4 -n gpu-operator``` command 

### 3. Information to [attach](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/) (optional if deemed irrelevant)
```
$ helm ls --all-namespaces
NAME               	NAMESPACE   	REVISION	UPDATED                                	STATUS  	CHART                     	APP VERSION
gpu-operator       	gpu-operator	1       	2020-05-10 13:18:46.667715068 +0000 UTC	deployed	gpu-operator-1.1.6        	1.1.6      
```

```
$ kubectl describe clusterpolicies --all-namespaces
Name:         cluster-policy
Namespace:    gpu-operator
Labels:       app.kubernetes.io/component=gpu-operator
              app.kubernetes.io/managed-by=Helm
Annotations:  meta.helm.sh/release-name: gpu-operator
              meta.helm.sh/release-namespace: gpu-operator
API Version:  nvidia.com/v1
Kind:         ClusterPolicy
Metadata:
  Creation Timestamp:  2020-05-10T13:18:47Z
  Generation:          1
  Resource Version:    1311916
  Self Link:           /apis/nvidia.com/v1/namespaces/gpu-operator/clusterpolicies/cluster-policy
  UID:                 e2982e73-f9af-4031-b322-e5c85080432b
Spec:
  Dcgm Exporter:
    Image:       dcgm-exporter
    Repository:  nvidia
    Version:     1.7.2-2.0.0-rc.5-ubuntu18.04
  Device Plugin:
    Image:       k8s-device-plugin
    Repository:  nvidia
    Version:     1.0.0-beta4
  Driver:
    Image:       driver
    Repository:  nvidia
    Version:     440.64.00
  Operator:
    Default Runtime:  docker
  Toolkit:
    Image:       container-toolkit
    Repository:  nvidia
    Version:     1.0.0-beta.0
Status:
  State:  notReady
Events:   <none>
```

**Comment by dkozlov** [comment]
probably related with the following https://github.com/kubernetes-sigs/controller-runtime/pull/675 issue

**Comment by raz-bn** [comment]
Any updates on this issue?
I am having the same problem

**Comment by nvjmayo** [comment]
Are you still seeing this issue in GPU Operator 1.1.7 or later? There have been namespace fixes between 1.1.6 and 1.1.7.

**Comment by pdmack** [comment]
https://docs.nvidia.com/datacenter/kubernetes/openshift-on-gpu-install-guide/index.html#openshift-gpu-troubleshooting
